exporters/metric/cortex/cortex_test.go

```diff
@@ -47,6 +64,9 @@ var validConfig = Config{
 	Client: http.DefaultClient,
 }
 
+var testResource = resource.New(kv.String("R", "V"))
+var mockTime int64 = time.Time{}.Unix()
```

Comment:
I see this `mockTime` defined and used in building out the `want` values, but how are the calls to `cmp.Equal(got, want)` being satisfied against `mockTime`?

Reply:
The metrictest package from the SDK generates records with time.Time{} for their start and end timestamp values. I'm using metrictest to make the checkpointsets in `testutil_test.go`, which is why time.Time{} is the correct comparison.